deliver/lib/deliver/options.rb

```diff
@@ -111,6 +111,15 @@ def self.available_options
                                      description: "Should the app be automatically released once it's approved?",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :auto_release_date,
+                                     env_name: "DELIVER_AUTO_RELEASE_DATE",
+                                     description: "Time to release when release on approval is used in milliseconds GMT",
```

Comment:
Not sure I fully understand what this description is supposed to mean. Could we change it to say something like "Date in milliseconds on which to automatically release pending approval"?

Reply:
yeah, I basically copy/pasted a comment from the app_version class. Your text is much nicer :)